src/shared/lib/hooks/useBanner.ts

```diff
@@ -0,0 +1,28 @@
+import { ToastProps } from '@app/app/ui/AppProvider/ToastProvider';
```

Comment:
According to [FSD](https://feature-sliced.design/docs/get-started/overview), lower layer are not allowed to access higher layers